enrichment_logbook_uploader/logbook_upload.py

```diff
@@ -109,6 +110,33 @@ def generate_payload(df, logbook_header_id):
         for date in skipped_dates:
             print(date)
 
+    get_logBook_response = requests.post(_ACTIVITY_ENRICHMENT_URL + "/LogBook/GetLogBook", headers=headers, data="logBookHeaderID="+logbook_header_id)
```

Comment:
Variable names should be lowercase in Python
```suggestion
    get_logbook_response = requests.post(_ACTIVITY_ENRICHMENT_URL + "/LogBook/GetLogBook", headers=headers, data="logBookHeaderID="+logbook_header_id)
```

Reply:
done